controllers/apps/transformer_cluster_component.go

```diff
@@ -143,6 +143,27 @@ func (t *clusterComponentTransformer) handleCompsUpdate(transCtx *clusterTransfo
 	return nil
 }
 
+func (t *clusterComponentTransformer) handleCompsDelete(transCtx *clusterTransformContext, dag *graph.DAG, deleteCompSet sets.Set[string]) error {
+	cluster := transCtx.Cluster
+	graphCli, _ := transCtx.Client.(model.GraphClient)
+	for compName := range deleteCompSet {
+		runningComp, err := getRunningCompObject(transCtx, cluster, compName)
+		if err != nil {
```

Comment:
the not found error should be ignored

Reply:
and components that have been deleted can be skipped

Reply:
> and components that have been deleted can be skipped

done